docs/MapColonies/vector/Services/geocoding/README.md

```diff
@@ -0,0 +1,1024 @@
+                "names": { "en": [ "JFK Airport" ], "fr": [ "Aeropuerto JFK" ], "default": [ "JFK" ], "display": "JFK Airport" },
+                "placetype": "transportation",
+                "sub_placetype": "airport",
+                "regions": [ { "region": "USA", "sub_region_names": [ "New York" ] } ]
+            }
+        }
+    ]
+}
+```
+</details>
+
+## Conversions
+
+:::tip
+Check [`Conversions request examples`](#conversions-1).
+:::
+
+You can convert WGS84 coordinates to 2 grids: Mapcolonies Control Grid/ US Army MGRS. In order to choose your target grid you pass `target_grid` query parameter.<br/>
+<details>
+    <summary>Conversion example</summary>
+
+```curl title="Request"
+curl --location '<geocoding_url>/lookup/coordinates?lat=52.57326537485767&lon=12.948781146422107&target_grid=control' \
+--header 'x-api-key: <x-api-key>' \
+--header 'x-user-id: <x-user-id>'
+```
+
+```json title="Response"
+{
+    "type": "Feature",
+    "geocoding": {
+        "version": "0.1.0",
+        "query": {
+            "lat": 52.57326537485767,
+            "lon": 12.948781146422107,
+            "target_grid": "control"
+        },
+        "response": {
+            "max_score": 1,
+            "results_count": 1,
+            "match_latency_ms": 0
+        }
+    },
+    "bbox": [ 12.93694771534361, 52.51211561266182, 13.080296161196031, 52.60444267653175 ],
+    "geometry": {
+        "type": "Polygon",
+        "coordinates": [ [ [ 12.93694771534361, 52.51211561266182 ],
+                [ 12.93694771534361, 52.60444267653175 ],
+                [ 13.080296161196031, 52.60444267653175 ],
+                [ 13.080296161196031, 52.51211561266182 ],
+                [ 12.93694771534361, 52.51211561266182 ] ] ]
+    },
+    "properties": {
+        "name": "BRN",
+        "tileName": "BRN",
+        "subTileNumber": [ "06", "97", "97" ]
+    }
+}
+```
+</details>
+
+## Common Query Params
+
+Almost all of our routes consists the same common query params: geo_context, geo_context_mode, limit & disable_fuzziness. <br/>
+* `geo_context`: via this param you can give the search engine the geo context of the search.
+* `geo_context_mode`: via this param you tell the search engine what to do with `geo_context`. You can filter results (which will result with every feature that matches the query and intersects with `geo_context` shape) or you can bias the results. So features that intersects with `geo_context` will appear first.
+* `limit`: by default, we will return our top 5 features that matches the query. You can change the limit and set it anything from 1 to 15 maximum results. We might return less than `limit`, but we limit the returned amount.
+* `disable_fuzziness`: fuzziness is on by default. If you want exact match, you may set `disable_fuzziness: true`. 
+
+## Usage
+:::caution
+**You will need an API token as part of the service authentication &nbsp;**
+:::
+
+:::caution
+**All routes require `x-api-key` and `x-user-id`.<br/> 
```

Comment:
After adding the link it isn't needed to explain this in detail (for the token).